### DISCUSSION TITLE
Can't add another github account bound email as secondary emails

### DISCUSSION THREAD

**Discussion by jbduzan** [community/community#129290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
I have two github account, one for work, one personal.
i would like to add the emails linked to those account to the other one as secondary emails, but i get an error when i try to do that.
I don't have other email address and don't intent to create more.

Is there a way to use those emails as secondary emails or at least to remove the warning displayed on every pages ?

Thanks

**Comment by ebndev** [comment]
Hi @jbduzan, thanks for being a part of the GitHub Community!

You are unable to connect emails that are linked to other accounts, as all emails listed connect commits to that specific account.

**Reply by rohankokkula** [reply]
Hey @ebndev 
I have a similar issue. I want to connect my personal email with my work email.
Does deactivating the github work account do the job?

**Comment by adarsh-site** [comment]
can you please share the warning that pops up.
